feature-libs/asm/styles/components/_customer-selection.component.scss

```diff
@@ -161,4 +159,170 @@
       }
     }
   }
+
+  @include forFeature('showSearchingCustomerByOrderInASM') {
+    form {
+      display: flex;
+      width: 100%;
+
+      @media (max-width: 575px) {
```

Comment:
these are styles from existing code. What about we use `rem` all in feature enabled styles? In this case, when we remove the feature flag, we can just remove the styles before line163, and there won't have any `px` in this file.